src/site/content/en/blog/idle-detection/index.md

```diff
@@ -163,6 +163,21 @@ controller.abort();
 console.log('IdleDetector is stopped.');
 ```
 
+### DevTools support
+
+You can emulate idle events in the Chrome DevTools, no need to actually idle around.
```

Comment:
Worth mentioning that this is Chrome 86+?

Reply:
Done.